### PR TITLE
Adds bitflag defs for flags_inv

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
Simple little PR that adds bitflag defs for the flags_inv var on /obj/item, so editing it in VV shows you the values as a toggle instead of the raw number.
[](https://i.imgur.com/viWEK2y.png)

## Why It's Good For The Game
Improved VV usability

### CODE REVIEW

**Comment by timothymtorres** [comment]
Unrelated question - if a component is initialized with a bitfield can this also be setup in a similar way?

**Comment by Mothblocks** [comment]
All the bitfield system does is check if the variable name is equivalent in VV, it's dumb enough to work for that.

File: code/_globalvars/bitfields.dm [hunk 1]
```diff
@@ -181,6 +181,22 @@ DEFINE_BITFIELD(item_flags, list(
...
+	"HIDESHOES" = HIDESHOES,
+	"HIDEMASK" = HIDEMASK,
+	"HIDEEARS" = HIDEEARS,
+	"HIDEEYES" = HIDEEYES,
+	"HIDEFACE" = HIDEFACE,
+	"HIDEHAIR" = HIDEHAIR,
+	"HIDEFACIALHAIR" = HIDEFACIALHAIR,
+	"HIDENECK" = HIDENECK,
+	"HIDEHEADGEAR" = HIDEHEADGEAR,
+	"HIDESNOUT" = HIDESNOUT,
```
Comment (Mothblocks) [inline code/_globalvars/bitfields.dm:197]: if you could abc-sort these please, i did this when i first made this file but not many people kept it up

Comment (Mothblocks) [reply code/_globalvars/bitfields.dm:197]: theres a vsc extension to bind this to f9 that i realize most people dont have and i should probably advertise

**Comment by Ryll-Ryll** [comment]
Done as requested

The "auto alphabetize lines" extension Mothblocks refers to is https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines by the way

**Pull request merged**